README.md

```diff
@@ -55,16 +63,18 @@ registry. The `$DEIS_REGISTRY` environment variable must point to a registry acc
 Kubernetes cluster. You may need to configure the Docker engines on your Kubernetes nodes to allow
 `--insecure-registry 192.168.0.0/16` (or the appropriate address range).
 
+When you want to test changes then commit the changes to your branch and run
+
 ```console
-$ make docker-build docker-push
+$ make deploy
 ```
 
-You'll want to modify the deis chart to use your custom image, then run `helm install` on the
-chart.
+This will build the required docker images and push up to the registry that was configure, update the Replication Controller
```

Comment:
I would reword this a bit:

> This will build the required docker images and push them to the registry that was configured, then update and recreate the Replication Controller.